pytorch_lightning/utilities/cli.py

```diff
@@ -109,7 +109,7 @@ def __init__(self, *args: Any, parse_as_dict: bool = True, **kwargs: Any) -> Non
         if not _JSONARGPARSE_AVAILABLE:
             raise ModuleNotFoundError(
                 "`jsonargparse` is not installed but it is required for the CLI."
-                " Install it with `pip install jsonargparse[signatures]`."
+                " Install it with `pip install -U jsonargparse[signatures]`."
             )
         super().__init__(*args, parse_as_dict=parse_as_dict, **kwargs)
```

Comment:
You won't be able to see the test failures if you don't test the new code-path (by removing `parse_as_dict`)

```suggestion
        super().__init__(*args, **kwargs)
```